### DISCUSSION TITLE
Can't install @react-spring/web

### DISCUSSION THREAD

**Discussion by susyabashti** [pmndrs/react-spring#1872] [Support]
Hi guys, I'm having trouble trying to install.

> npm ERR! code ERESOLVE
> npm ERR! ERESOLVE unable to resolve dependency tree
> npm ERR! 
> npm ERR! While resolving: test@0.1.0
> npm ERR! Found: react@18.0.0
> npm ERR! node_modules/react
> npm ERR!   react@"^18.0.0" from the root project
> npm ERR! 
> npm ERR! Could not resolve dependency:
> npm ERR! peer react@"^16.8.0  || ^17.0.0" from @react-spring/web@9.4.4
> npm ERR! node_modules/@react-spring/web
> npm ERR!   @react-spring/web@"9.4.4" from the root project
> npm ERR! 
> npm ERR! Fix the upstream dependency conflict, or retry
> npm ERR! this command with --force, or --legacy-peer-deps
> npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

This is what I get when trying to install. does anyone knows why?

**Comment by joshuaellis** [comment]
our current stable version does not support react@18, instead I would use the most recent beta – `9.4.5-beta.1`
